### ISSUE TITLE
Show Columns funktioniert nicht

### ISSUE THREAD

**Issue by McCompexpert** [EkoDondav/rehaklex#22]
funktioniert auch noch nicht.

**Comment by EkoDondav** [comment]
geht nur, wenn du sie zuvor mit klick auf table col ausblendest. besser wäre natürlich, nur ausgeblendeten cols unten in show colums anzuzeigen...

**Comment by McCompexpert** [comment]
Ja, das geht. Was bewirkt das Klicken auf "Own"? Soll das nicht nur die Zeilen anzeigen, wo ich eine Position habe?

**Comment by EkoDondav** [comment]
Own zeigt alle sechs own-spalten wieder an, falls zuvor per klick-auf-einen-der spalten-titel) unterdrückt...

**Comment by McCompexpert** [comment]
Ok
